### PR TITLE
Added speexdsp.

### PR DESCRIPTION
**Pull request by Rinnegatamante**
Added speexdsp.

### CODE REVIEW

**Comment by xyzz** [comment]
You need to add it to travis_packages.sh

**Comment by Rinnegatamante** [comment]
Done

File: speexdsp/VITABUILD [hunk 1]
```diff
@@ -0,0 +1,17 @@
+pkgname=speexdsp
+pkgver=1.2rc3
+pkgrel=1
+url="http://xiph.org/"
+source=("http://downloads.xiph.org/releases/speex/speexdsp-1.2rc3.tar.gz")
```
Comment (xyzz) [inline speexdsp/VITABUILD]: Use ${pkgver} in the URL

**Comment by Rinnegatamante** [comment]
@xyzz Done

**Pull request merged**